openvdb_points_houdini/houdini/SOP_OpenVDB_Points.cc

```diff
@@ -593,11 +593,23 @@ newSopOperator(OP_OperatorTable* table)
         .setDefault("points")
         .setHelpText("Output grid name."));
 
+    parms.add(hutil::ParmFactory(PRM_TOGGLE, "automatic", "Automatic Voxel Size")
+        .setDefault(PRMzeroDefaults)
+        .setHelpText("*BETA* When enabled, automatically attempts to calculate a "
+                     "target transform of an input Houdini Point set for best "
+                     "representing the points in a PointDataGrid."));
+
     parms.add(hutil::ParmFactory(PRM_FLT_J, "voxelsize", "Voxel Size")
         .setDefault(PRMpointOneDefaults)
         .setHelpText("The desired voxel size of the new VDB Points grid.")
         .setRange(PRM_RANGE_RESTRICTED, 0, PRM_RANGE_UI, 5));
 
+    parms.add(hutil::ParmFactory(PRM_INT_J, "pointspervoxel", "Points Per Voxel")
+        .setDefault(8)
+        .setHelpText("The average amount of points per voxel to aim for in the new "
```

Comment:
"The number of points per voxel to use as the target for automatic voxel size computation." ?